test/e2e/tests/account/account-modal-account-type-change.spec.ts

```diff
@@ -12,20 +12,6 @@ import { WINDOW_TITLES, withFixtures } from '../../helpers';
 import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
 import { mockEip7702FeatureFlag } from '../confirmations/helpers';
 
-// Function is needed to increase the gas limit for upgrade account confirmation
-// There is apparently an issue with Anvil network that prevents correct estimation of gas limit for upgrade.
-const increaseGasLimit = async (driver: Driver) => {
-  await driver.clickElement('[data-testid="edit-gas-fee-icon"]');
-  await driver.clickElement('[data-testid="edit-gas-fee-item-custom"]');
-
-  await driver.clickElement('[data-testid="advanced-gas-fee-edit"]');
-  await driver.fill('[data-testid="gas-limit-input"]', '50000');
-
-  // Submit gas fee changes
-  await driver.clickElement({ text: 'Save', tag: 'button' });
-  await driver.clickElement({ text: 'Advanced' });
-};
-
```

Comment:
this piece was not using page objects. Also, we don't need this as a function.
I've added a 1 line change now that we can use POM method below